jscomps/PrivateDataManager.js

```diff
@@ -54,13 +43,15 @@ PrivateDataManager.prototype = {
         break;
       }
       case "cookies": {
-        this.cookieManager.removeAll();
-        debug("Cookies removed");
+        this.clearData(Ci.nsIClearDataService.CLEAR_COOKIES);
         break;
       }
       case "cache": {
-        this.clearCaches()
-        debug("Cache cleaned");
+        this.clearData(Ci.nsIClearDataService.CLEAR_ALL_CACHES);
+        break;
+      }
+      case "site-data": {
```

Comment:
At embedlite-components side maybe it's ok to keep "cookies" and "site data" separated but in the user interface I think we should combine these. The "site data" is not very clear for end user.

Reply:
Let's clear cookies and site data at once as firefox does it.